apps/payments/next/app/page.tsx

```diff
@@ -42,7 +49,7 @@ export default function Index() {
               <h2>123Done Pro - Monthly</h2>
               <Link
                 className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
-                href="/en/123donepro/monthly/new"
+                href={`/${locale}/123donepro/monthly/new`}
```

Comment:
[non-blocking] what's the behavior when the user is routed to a locale that we don't support?